en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,191 @@
+
+  - **Standard:** Approximately 6 months, until the next minor release is performed
+  - **Extended:** One year
+
+### Partial support
+
+At the end of the *full support* phase, a Silverstripe CMS version transitions to the *partial support* phase.
+
+- **Security fixes:** Partial – only high and critical impact security vulnerabilities (those with a CVSS score of 7.0 or above) are addressed
+- **Bug fixes:** No
+- **New features:** No
+- **Enhancements:** No
+- **Support length:**
+
+  - **Standard:** 6 months
+  - **Extended:** Approximately one year, extending until the subsequent major release (e.g. CMS 5.4 will remain in partial support until CMS 7.0 is released)
+
+### End-of-life
+
+Once the *partial support* phase for a Silverstripe CMS version has ended, it goes *end-of-life* This means the release is no longer supported.
+
+- **Security fixes:** No
+- **Bug fixes:** No
+- **New features:** No
+- **Enhancements:** No
+
+> [!IMPORTANT]
+> Users on end-of-life versions are strongly encouraged to upgrade to a supported version
+
+## Patch releases
+
+Patch releases are small code releases for individual modules such as `silverstripe/framework`. They can happen at any time and are not part of a broader Silverstripe CMS release. They are the delivery mechanism to fulfill the support commitments for Silverstripe CMS versions that are supported.
+
+Patch releases should not make any public API changes. Because of this restriction, typically only bug and security fixes will be shipped in them.
+
+## Security patch windows
+
+Security fixes differ from regular bug fixes due to their sensitive nature. Security vulnerabilities are handled responsibly and not publicly disclosed before a patch is available.
+
+Patches containing security fixes release at pre-defined intervals to make it easier to plan Silverstripe CMS project upgrades.
+
+When a vulnerability is [confidentially reported](/contributing/issues_and_bugs/#reporting-security-issues) to the Silverstripe CMS development team and we are confident that the vulnerability is not actively being exploited in the wild, we will wait until the next available security patch window to disclose it and release a patch.
+
+Security patch windows are scheduled to line up with our expected minor releases. Our security patch windows are in:
+
+- January
+- April
+- July
+- October
+
+Any security fixes that are shipped at the same time as a new Silverstripe CMS release will be included in that release.
+
+## PHP support commitments
+
+The Silverstripe CMS release cycle is built around these assumptions:
+
+- New PHP releases are shipped at the end of November on an annual basis.
+- PHP releases are in full support for two years, followed by two years of support for critical security fixes only.
+
+At launch, a Silverstripe CMS major release supports all PHP versions in full support. PHP versions in limited support are not supported at launch by the new Silverstripe CMS major version series.
```

Comment:
```suggestion
At launch, a Silverstripe CMS major release supports all PHP versions in full support. PHP versions in limited support are not supported by the new Silverstripe CMS major version series.
```
Just removing the "at launch" bit - we wouldn't add backwards support after launch i.e. we won't add PHP 7.4 support for CMS 6.